docs/SRS-Volere/SRS.tex

```diff
@@ -252,22 +252,52 @@ \subsection{Products That Can Be Copied}
 
 \section{New Problems}
 \subsection{Effects on the Current Environment}
-\lips
+\begin{enumerate} [{}1.]
+  \item The application should not create redundant workload for tasks already carried out by existing solutions.
+\end{enumerate}
```

Comment:
This section is like how will this system affect the work done at the City

Reply:
True, but the New Problems section is to define what the system should not do, I was not sure what it should not do other than not create extra work for the client. Looking at most of the example SRS given in the GitLab, and those submitted by teams in the past, these sections were left blank (N/A) or had a sentence or two.